eventhandlers/action_runners/actions/HuggingFaceZeroShot.py

```diff
@@ -34,78 +39,87 @@ def test_execute_action(self, session, file_id, connection_id):
         pass
 
     def execute_action(self, session, do_save_annotations=True) -> bool:
-        print("Hugging face has been executed")
-        return
-        """
-                    Creates a task from the given file_id in the given task template ID.
-                :return:
-                """
-
+        event_name = self.action.trigger_data.get('event_name')
         file_id = self.event_data['file_id']
-        if not file_id:
-            logger.warning(f'Action has no file_id Stopping execution')
-            return
-    
+        project_id = self.action.config_data['project_id']
+        group_id = self.action.config_data['group_id']
+
+        task = None
+        task_id = None
+
+        if event_name == 'task_created':
```

Comment:
refactor to generic for actions?
is task id not trusted here?

Reply:
What do you mean by "not trusted"? If you mean that it's optional - then yes, if the trigger is input_file_uploaded - then action will be executed without task_id